doc/start.rst

```diff
@@ -39,3 +39,7 @@ Here is a brief example of how to use the ``GLM()`` class.
 
     # score the model on test data
     deviance = glm.score(Xtest, ytest)
+
+``GLM()`` class is also designed to be compatible with ``scikit-learn``. 
```

Comment:
"perform" or "do" instead of "performing"

Reply:
thanks for the catch! fixed!